Build ispell as static-pie executable (#15)
unikraft/static-pie-apps: ispell/README.md

# Build ispell as static PIE

> Build the [ispell](https://www.nginx.com/) tool static PIE ELF running on Linux.

Reviewer: ```suggestion
Build the [ispell](https://www.cs.hmc.edu/~geoff/ispell.html) tool as a static PIE ELF running on Linux.
```

## Requirements

> Make sure the following packages are installed:
> * gcc >= 8 (required for the `-static-pie` linking option)
> * GNU Make

Reviewer: ```suggestion
Make sure the following packages are installed:

* `gcc` >= 8 (required for the `-static-pie` linking option)
* `GNU Make`
```

For Ubuntu, run the command below to install all requirements:

```console
$ sudo apt install build-essential
```

## Build

> The `ispell` static PIE ELF is located in the current folder.

Reviewer: ```suggestion
The `ispell` static PIE ELF is located in the current directory.
```
